src/runtime/node/io.js

```diff
@@ -2,6 +2,7 @@
 
 import { readFile } from 'fs';
 import { L10nError } from '../../lib/errors';
+import 'string.prototype.endswith';
```

Comment:
FYI: this is required for server side node since it doesn't ship with an `String.prototype.endsWith`. 

It's also worth mentioning, that we may want to include this in our other web distributions since not all browsers are shipping `String.prototype.endsWith` either.